README.md

```diff
@@ -1,10 +1,41 @@
-# Infinite Industries 
+# Infinite Industries
+<img src="./docs/doc-splash.png">
```

Comment:
If you want to do this in markdown as well you can do ![My Alt Text](./docs/doc-splash.png)

In the case of images I'm not sure it matters as much which way we do it because neither is exactly super readable, but might be nice to stay consistent